src/chrome/content/rules/Wikimedia.xml

```diff
@@ -64,10 +67,11 @@
 		<!-- mismatch on tools.wikimedia.de -->
 		<exclusion pattern="^http://tools\.wikimedia\.de/" />
 			<test url="http://tools.wikimedia.de/" />
+		<exclusion pattern="^http://civicrm\.wikimedia\.de/" /> 
```

Comment:
That's because of an incomplete cert chain. Could you document that?

Reply:
@J0WI Done, and the build finally passed.